ExaltedApotheosisPreReqs.cs

```diff
@@ -0,0 +1,59 @@
+    //cs_include Scripts/CoreBots.cs
+    //cs_include Scripts/CoreFarms.cs
+    using Skua.Core.Interfaces;
+
+    public class ExaltedApotheosisPreReqs
+    {
+        public IScriptInterface Bot => IScriptInterface.Instance;
+        public CoreBots Core => CoreBots.Instance;
+        public CoreFarms Farm = new();
+
+        public void ScriptMain(IScriptInterface bot)
+        {
+            Core.SetOptions();
+
+            PreReqs();
+
+            Core.SetOptions(false);
+        }
+
+        public void PreReqs()
+        {

```

Comment:
a general comment here about Core.Buy // Adv.Buy
it will internally handle merge requirements and buy them in order to buy your exact item
for example of Weapon 4 requires that the player have weapon 2 and 3 and 500k voucher
and u iserted Core.BuyItem ("Map", ShopId, "Weapon 4");
then it will autamatically buy weapon 2 and 3 and 500k voucher from same merge (if the player have their respective requirement)
idk how u gonna do this in here and i cba to look into wiki to give u a hint

Reply:
The weapons require 150 misc of a misc item then a couple stacks of boss misc drops on top to merge so I doubt it would know where to farm them. That's a sick option to use tho for more simple stuff thx.